packages/apollos-data-connector-postgres/src/follows/dataSource.js

```diff
@@ -61,11 +61,19 @@ class Follow extends PostgresDataSource {
       // If a request already exists that was denied, update it as a new request.
       return existingFollow.update({ state: FollowState.REQUESTED });
     }
+
+    let initialState = FollowState.REQUESTED;
+
+    // Automatically accept requests to follow suggested people
+    const suggested = await this.getStaticSuggestedFollowsForCurrentPerson();
+    if (suggested.some((s) => s.id === parseGlobalId(followedPersonId).id))
```

Comment:
I could go either way on this one :) the id is parsed already above, but it's nice to have the clarity of what id is. 
```suggestion
    if (suggested.some((s) => s.id === id)
```